### ISSUE TITLE
A11y_Azure Tools Storage Explorer_ Toolbar_ Toolbar controls_Screenreader: Screen reader is not announcing the “copied” information when user activates the copy button present on blob container toolbars

### ISSUE THREAD

**Issue by Jtith** [microsoft/AzureStorageExplorer#6372]
**GitHub Tags:**
#A11yMAS; #A11yTCS; #Mac; #DesktopApp; #A11ySev3; #BM_AzureToolsStorageExplorer_Mac_Dec2022; #Azure Tools Storage Explorer;  #WCAG1.3.1; #Voiceover;  #FTP; #DesktopApp;

### Actual results:
Screen reader is not announcing the “copied” information when user activates the copy button present on blob container toolbars.
### Expected results:
Screen reader should announce the “copied” information when user activates the copy button by pressing on enter.
### Environment Details:
Application: Microsoft Azure Storage Explore

### OS Details
Mac details: 
### Repro Steps:
1. Launch Storage Explorer
2. Tab to "Open Connect dialog" pane and Sign into Azure.
3. Tab to "Manage Accounts" pane and select the subscriptions you will use. Once selected, click on the “Apply" button.
4. Tab to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5. Once completed, Tab to "Upload" button and upload a blob/file by using “Upload Files" control.
6.  Turn on Voiceover.
7. Navigate to Toolbar and tab to Copy button and press on enter to activate
8. And check whether the screen reader is announcing the copied information when user activates the copy button or not.

### User Impact:
Screen reader user will not know about copied information If it’s not announced.

**Comment by Jtith** [comment]
i have verified in the latest environment, issue still repro. Adding attachment for reference.
[Issue still repro (2).zip](https://github.com/microsoft/AzureStorageExplorer/files/10499280/Issue.still.repro.2.zip)

**Comment by Jtith** [comment]
This is a sev3 issue you can fix the bug or add it in your backlog

**Comment by Mana-swi** [comment]
#a11ydb-status-not-conveyed;
